tox.ini

```diff
@@ -9,7 +9,7 @@ commands =
     # Using pytest-cov instead of using coverage directly leaves a bunch of
     # .coverage.$HOSTNAME.#.# files lying around for some reason
     coverage erase
-    coverage run -m pytest -v {posargs} --doctest-modules dandi
+    coverage run -m pytest -v {posargs} dandi
```

Comment:
why no doctest modules any more?  that could leave that example you created for dandiapi without even smoke testing

Reply:
Because, due to the increased number of published Dandisets, it now takes six minutes to run the doctest.  We still test the doctest as part of the scheduled CI test.

Reply:
ah, ok -- fair enough. Thanks!